osu.Game.Rulesets.Taiko/UI/TaikoPlayfieldAdjustmentContainer.cs

```diff
@@ -51,10 +55,38 @@ protected override void Update()
             // Limit the maximum relative height of the playfield to one-third of available area to avoid it masking out on extreme resolutions.
             relativeHeight = Math.Min(relativeHeight, 1f / 3f);
 
-            Y = base_position;
-
             Scale = new Vector2(Math.Max((Parent!.ChildSize.Y / 768f) * (relativeHeight / base_relative_height), 1f));
             Width = 1 / Scale.X;
         }
+
+        public double ComputeTimeRange()
+        {
+            float currentAspect = Parent!.ChildSize.X / Parent!.ChildSize.Y;
+
+            if (LockPlayfieldAspectRange.Value)
+                currentAspect = Math.Clamp(currentAspect, MINIMUM_ASPECT, MAXIMUM_ASPECT);
+
+            // in a game resolution of 1024x768, stable's scrolling system consists of objects being placed 600px (widthScaled - 40) away from their hit location.
+            // however, the point at which the object renders at the end of the screen is exactly x=640, but stable makes the object start moving from beyond the screen instead of the boundary point.
```

Comment:
> the point at which the object renders at the end of the screen is exactly x=640, but stable makes the object start moving from beyond the screen instead of the boundary point

no reference for this